mne/viz/utils.py

```diff
@@ -612,6 +616,13 @@ def _radio_clicked(label, params):
     params['plot_fun']()
 
 
+def _get_active_radiobutton(radio):
+    """Helper to find out active radio button."""
+    # XXX: In mpl 1.5 you can do: fig.radio.value_selected
```

Comment:
This returns an index and ``value_selected`` returns the text, so I would have to map it to the index. I think it's cleaner like this.